src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -236,6 +240,9 @@ def is_alive(self) -> bool:
         return True
 
     def get_forwards(self) -> List[Forward]:
+        if self.private_ip is None:
```

Comment:
We call `get_forwards()` on `save_proxy_config()` and `is_used()`. Is there ever a time we want a `Proxy` object, but it is okay for `self.private_ip` to be `None`?

If not, would it be better to raise an error when we get the `private_ip` in the first place?

Reply:
When the Proxy object is initially created, but we're waiting for Azure to allocate the VMs, NICs, and IPs, the private_ip can be none.